### PR TITLE
fix: backward compatible runtime env vars for client plugins

### PR DESCRIPTION
**Pull request by aldarund**
### 🔗 Linked issue

#9

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

allows access env vars in bridge via $store in plugins on client

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/runtime/nitro/renderer.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default eventHandler(async (event) => {
     event,
     req: event.req,
     res: event.res,
-    runtimeConfig: { private: config, public: { public: config.public, app: config.app } },
+    runtimeConfig: { private: config, public: { ...config.public, public: config.public, app: config.app } },
```
Comment (pi0) [inline src/runtime/nitro/renderer.ts:93]: It should be handled by the runtime proxy.

Comment (aldarund) [reply src/runtime/nitro/renderer.ts:93]: As i understand runtime proxy is a config object itself that comes from composable useRuntimeConfig , but here its set values as a new object with public and app path, which  nullify that proxy so it leads to no props at root like nuxt 2 expect

**Comment by danielroe** [comment]
@aldarund Would you check whether https://github.com/nuxt/bridge/pull/552 resolves the issue for you?

**Comment by danielroe** [comment]
Closing as I think the root issue is solved, but please ping me if not 🙏

**Pull request closed without merging**